src/main/scala/io/iohk/praos/domain/Environment.scala

```diff
@@ -1,7 +1,11 @@
 package io.iohk.praos.domain
 
+import io.iohk.praos.crypto.Seed
+
 case class Environment(initialStakeDistribution: StakeDistribution,
                        epochLength: Int,
+                       k: Int,
```

Comment:
What is this `k`? Maybe we can rename this variable or add a comment on what it's purpose is?